test/common.py

```diff
@@ -4,9 +4,8 @@
 import sys
```

Comment:
This looks good, but did you test the tooling works at all?

Reply:
No, I haven't. What is the best way to run those tests?

I just notices that the following tests still have the old imports that won't work with the updated LiteX.

```test_sdram_bist.py:3:16:from litex.soc.tools.remote import RemoteClient
  test_etherbone.py:7:16:from litex.soc.tools.remote import RemoteClient
  test_alignment.py:4:16:from litex.soc.tools.remote import RemoteClient  
  test_hdmi_out.py:3:16:from litex.soc.tools.remote import RemoteClient   
  test_analyzer.py:4:16:from litex.soc.tools.remote import RemoteClient    
  test_delays.py:4:16:from litex.soc.tools.remote import RemoteClient       
  test_sdram.py:2:16:from litex.soc.tools.remote import RemoteClient       
  test_xadc.py:1:16:from litex.soc.tools.remote import RemoteClient         
  test_ddr3.py:2:16:from litex.soc.tools.remote import RemoteClient         
  test_leds.py:2:16:from litex.soc.tools.remote import RemoteClient          
  test_dump.py:2:16:from litex.soc.tools.remote import RemoteClient  
```